ROADMAP.md

```diff
@@ -0,0 +1,98 @@
+Ansible Roadmap
+=============
+This document is now the location for published Ansible Core roadmaps.     
+
+The roadmap will be updated by version. Based on team and community feedback, an initial roadmap will be published for a major or minor version (2.0, 2.1).  Subminor versions will generally not have roadmaps published.
+
+This is the first time we've published this and asked for feedback in this manner.  So feedback on the roadmap and the new process is quite welcome.  We are aiming for further transparency and better inclusion of both community desires and submissions.  
+
+These are our *best guess* roadmaps based on our own experience and based on requests and feedback from the community.  There are things that may not make it on due to time constraints, lack of community maintainers, etc.  And there may be things we missed, so each roadmap is published both as an idea of what is upcoming in Ansible, and as a medium for seeking further feedback from the community. Here are the good places for you to submit feedback:
+
```

Comment:
Suggestion:  "These roadmaps are *best guesses* based on Ansible's experiences as well as being based on requests and feedback from the community."  and  "And there may be things that were missed, so each..."